kroma-validator/guardian.go

```diff
@@ -486,8 +486,8 @@ func (g *Guardian) tryConfirmRequestDeletionTx(event *bindings.SecurityCouncilDe
 	return nil
 }
 
-func (g *Guardian) checkConfirmCondition(transactionId *big.Int, outputIndex *big.Int) (bool, error) {
-	outputFinalized, err := g.IsOutputFinalized(g.ctx, outputIndex)
+func (g *Guardian) CheckConfirmCondition(ctx context.Context, transactionId *big.Int, outputIndex *big.Int) (bool, error) {
+	outputFinalized, err := g.isOutputFinalized(ctx, outputIndex)
```

Comment:
please add a sub context with network timeout

Reply:
In the function `isOutputFinalized`, there is a declaration of new context with timeout.

Reply:
oh, right. I missed that.